SpatialGDK/Source/SpatialGDKFunctionalTests/Public/SpatialFunctionalTest.h

```diff
@@ -140,6 +140,10 @@ class SPATIALGDKFUNCTIONALTESTS_API ASpatialFunctionalTest : public AFunctionalT
 	template <class T>
 	T* SpawnActor(const FActorSpawnParameters& SpawnParameters = FActorSpawnParameters(), const bool bRegisterAsAutoDestroy = true);
 
+	template <class T>
+	T* SpawnActor(const FVector& Location, const FRotator& Rotation, const FActorSpawnParameters& SpawnParameters = FActorSpawnParameters(),
+				  const bool bRegisterAsAutoDestroy = true);
```

Comment:
```suggestion
	T* SpawnActor(const FVector& Location = FVector::ZeroVector, const FRotator& Rotation = FRotator::ZeroRotator, const FActorSpawnParameters& SpawnParameters = FActorSpawnParameters(),
				  const bool bRegisterAsAutoDestroy = true);
```

Reply:
unfortunately not possible as would be ambiguous with the above method, included the rotation default though

Reply:
Thinking about it I think you want to keep the above overload but just remove the default arg. That way you can still `SpawnActor` with `FActorSpawnParameters` as its first parameter which is ergonomic.

Reply:
I really want c++20 named args already..

Reply:
hmm, I see your point. I guess at this point it doesn't matter whether the location is defaulted or the spawnparams are as either way you get a method that does what you want when given no args.

I'm leaning towards keeping spawnparams defaulted as there are one or two cases where bRegisterAsAutoDestroy needs to be set, so less args need to be passed

Reply:
just create an overload that just takes in `bRegisterAsAutoDestroy` ?

Reply:
I would go for one full version with everything defaulted and selected versions with _1_ argument non defaulted that calls the full one.

Reply:
actually yeah that's a good approach, will do

Reply:
added!